woocommerce/functions.php

```diff
@@ -47,7 +47,7 @@ function siteorigin_north_woocommerce_enqueue_styles( $styles ){
 		);
 	}
 
-	if( function_exists('is_woocommerce') && is_woocommerce() && siteorigin_setting( 'responsive_disabled' ) ) {
+	if( function_exists('is_woocommerce') && siteorigin_setting( 'responsive_disabled' ) ) {
```

Comment:
As we are using the `woocommerce_enqueue_styles` filter hook, the function should run only when WooCommerce is activated. 

So removing `function_exists('is_woocommerce')` should be ok.